docs/pages/recent-releases.mdx

```diff
@@ -0,0 +1,364 @@
+    </tr>
+    <tr>
+      <td>[`Tabs`](https://60a1e4a102f0cb003b5d19d6-xvoxshqkig.chromatic.com/?path=/docs/components-tabs-tabs-deprecated--docs)</td>
+      <td>[`Tabs`](?path=/docs/components-tabs-api-specification--docs)</td>
+      <td>Manual change required</td>
+      <td>[Migration guide](https://60a1e4a102f0cb003b5d19d6-xvoxshqkig.chromatic.com/?path=/docs/components-tabs-migration-guide--docs)</td>
+    </tr>
+    <tr>
+      <td>[`Tooltip`](https://60a1e4a102f0cb003b5d19d6-xvoxshqkig.chromatic.com/?path=/docs/components-tooltip-tooltip-deprecated--docs)</td>
+      <td>[`Tooltip`](?path=/docs/components-tooltip-api-specification--docs)</td>
+      <td>Feasibility TBD</td>
+      <td>[Migration guide](https://60a1e4a102f0cb003b5d19d6-xvoxshqkig.chromatic.com/?path=/docs/components-tooltip-migration-guide--docs)</td>
+    </tr>
+  </tbody>
+</table>
+
+### Other components being removed
+
+<table>
+  <thead>
+    <tr>
+      <th>Component</th>
+      <th>Recommendation</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+        [`Skirt`](https://60a1e4a102f0cb003b5d19d6-xvoxshqkig.chromatic.com/?path=/docs/components-skirt-deprecated--docs)
+      </td>
+      <td>Remove any usages</td>
+    </tr>
+    <tr>
+      <td>
+        [`SkirtCard`](https://60a1e4a102f0cb003b5d19d6-xvoxshqkig.chromatic.com/?path=/docs/components-skirt-deprecated--docs)
+      </td>
+      <td>Use a regular `Card` component</td>
+    </tr>
+    <tr>
+      <td>
+        [`SplitButton`](https://60a1e4a102f0cb003b5d19d6-xvoxshqkig.chromatic.com/?path=/docs/components-splitbutton-deprecated--docs)
+      </td>
+      <td>Use a `Button` and a `Menu` component (story to come)</td>
+    </tr>
+  </tbody>
+</table>
+
+### Renamed components
+
+Imports from the new names are now available, and support for the old names was removed in v2.
+
+<table>
+  <thead>
+    <tr>
+      <th>Old name</th>
+      <th>New name</th>
+      <th>Codemod</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>`next/Select`</td>
+      <td>`SingleSelect`</td>
+      <td>Available</td>
+    </tr>
+    <tr>
+      <td>`LikertScaleLegacy`</td>
+      <td>`LikertScale`</td>
+      <td>Available</td>
+    </tr>
+    <tr>
+      <td>`TitleBlockZen`</td>
+      <td>`TitleBlock`</td>
+      <td>Available</td>
+    </tr>
+  </tbody>
+</table>
+
+### Deprecated props being removed
+
+<table>
+  <thead>
+    <tr>
+      <th>Component</th>
+      <th>Prop</th>
+      <th>Codemod</th>
+      <th>Reason</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td colSpan={4}>`mood` and `variant`</td>
+    </tr>
+    <tr>
+      <td>`BrandMoment`</td>
+      <td>`mood`</td>
+      <td>`migrateBrandMomentMoodToVariant`</td>
+      <td>Use `variant` instead</td>
+    </tr>
+    <tr>
+      <td>`Card`</td>
+      <td>`variant`</td>
+      <td>`migrateCardVariantToColor`</td>
+      <td>Use `color` instead</td>
```

Comment:
Everything else seems to use `variant`, how come `Card` uses `color` for this?

Reply:
They're are literal colour selections vs the variant being more around the intent of something "primary", "secondary" etc